.github/workflows/code-coverage.yml

```diff
@@ -0,0 +1,43 @@
+name: Code Coverage
+
+on:
+  push:
+    branches: ["master"]
+  pull_request:
+    branches: ["master"]
+
+jobs:
+  coverage:
+    name: Code Coverage
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Setup PHP with Xdebug
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: "8.1"
+          coverage: xdebug
+
+      - name: Create Folder and File for Vendor Path
+        run: mkdir -p app/etc && cd app/etc && touch vendor_path.php
```

Comment:
Do we need this for WooCommerce? I think these are specific to Magento.

Reply:
Ahh No need. I will remove it. I over copy.